### PR TITLE
signup/login flow for seeker

### PR DESCRIPTION
**Pull request by ankitmishra03**
- signup/login flow for seeker
- backpress handled at various places

### CODE REVIEW

File: client/src/components/Navbar/NavbarElements.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+// import { FaBars } from "react-icons/fa";
```
Comment (bhavneet2000) [inline client/src/components/Navbar/NavbarElements.js:1]: Delete file

File: client/src/App.js [hunk 2]
```diff
@@ -16,18 +16,27 @@ import AddSeeker from "./pages/AddSeeker";
...
+  const isDashboardPage =
+    location.pathname.includes("/dashboard") ||
+    (isLoggedIn && location.pathname === "/");
   return (
     <div className="w-screen h-screen bg-white flex flex-col">
       {!isDashboardPage && <NavbarCmp />}
       <Routes>
-        <Route path="/" element={<Home />} />
+        <Route path="/" element={isLoggedIn ? <Dashboard /> : <Home />} />
+        <Route path="/dashboard" element={<Dashboard />} />
```
Comment (bhavneet2000) [inline client/src/App.js:38]: What is diff between /dashboard and /dashboard/employee ?

File: client/src/pages/Home.js [hunk 3]
```diff
@@ -38,6 +38,7 @@ const Home = () => (
           </p>
 
           <Button />
+         {/* <Button2 /> */}
```
Comment (bhavneet2000) [inline client/src/pages/Home.js:41]: remove

File: client/src/App.js (hunk 2 above)
Comment (ankitmishra03) [reply client/src/App.js:38]: it was earlier needed, have removed it now.

**Review by bhavneet2000 — APPROVED** [review]

**Pull request merged**